src/games/planetwars/visualizer/controls.js

```diff
@@ -1,12 +1,12 @@
 const d3 = require('d3');
 const Config = require('./config');
-const Visualizer = require('./visualizer');
 
 class Controls {
-  constructor(visualizer) {
-    this.visualizer = visualizer;
+  constructor() {
     this.mod = 3;
     this.updateSpeed(Config.speed_mods[this.mod]);
+
+    // Set visibility for some buttons
     d3.select('#unhide').classed('invisible', true);

```

Comment:
Make this consistent with other button hiding. Maybe abstract into something that conveys the essential idea better.